lib/wibox/layout/align.lua

```diff
@@ -200,7 +278,6 @@ end
 -- @property third
 -- @tparam widget third
 -- @propemits true false
-
```

Comment:
We don't usually add empty lines between doc comments and the things they document. So if you do want to make this consistent, you should remove the empty line at `set_first` instead.